src/App.jsx

```diff
@@ -1,13 +1,19 @@
 import * as React from "react";
-import TodoList from "./TodoList.jsx";
-import AddTodoForm from "./AddTodoForm.jsx";
+import TodoList from "./components/TodoList";
+import AddTodoForm from "./components/AddTodoForm.jsx";
 import { useState, useEffect } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import styles from "./App.module.css";
 
 // const title = 'React';
 
 function App() {
   let onLoadList = JSON.parse(localStorage.getItem("savedTodoList"));
 
+  useEffect(() => {
+    document.body.style.backgroundColor = "#3657EB";
```

Comment:
Hey Grant, why are you changing the body's background color here in the useEffect hook as opposed to using CSS?